src/main/java/meteordevelopment/meteorclient/systems/modules/render/BlockSelection.java

```diff
@@ -65,6 +76,15 @@ public BlockSelection() {
     private void onRender(Render3DEvent event) {
         if (mc.crosshairTarget == null || !(mc.crosshairTarget instanceof BlockHitResult result)) return;
 
+        BlockPos playerEyePos = new BlockPos(mc.player.getEyePos());
+        int crossX = (int) Math.floor(mc.crosshairTarget.getPos().x);
+        int crossY = (int) Math.floor(mc.crosshairTarget.getPos().y);
+        int crossZ = (int) Math.floor(mc.crosshairTarget.getPos().z);
+
+        if(hideInside.get() && result.isInsideBlock()) {
```

Comment:
Can you follow our standards and put a space between `if` and `(`. Also the return could be on the same line but that's not important.